### ISSUE TITLE
Cannot find module '@rootly/backstage-plugin/src/components/Entity' from 'components/catalog/EntityPage.tsx'

### ISSUE THREAD

**Issue by mcncl** [rootlyhq/backstage-plugin#1]
The plugin isn't building correctly, it has no `dist` or `src` folder, which contains the plugin itself, leading to an error seen below. I've also attached a screenshot of the plugin dir, compared to the Buildkite one. 

```shell


The Backstage CLI jest transformModules option is no longer used and will be ignored. All modules are now always transformed.
--
  | FAIL  src/App.test.tsx
  | ● Test suite failed to run
  |  
  | Cannot find module '@rootly/backstage-plugin/src/components/Entity' from 'components/catalog/EntityPage.tsx'
  |  
  | Require stack:
  | components/catalog/EntityPage.tsx
  | App.tsx
  | App.test.tsx
  |  
  | 68 \|   RELATION_PROVIDES_API
  | 69 \| } from '@backstage/catalog-model'
  | > 70 \| import { RootlyOverviewCard } from '@rootly/backstage-plugin/src/components/Entity';
  | \|               ^
  | 71 \| import { RootlyIncidentsPage } from '@rootly/backstage-plugin';
  | 72 \|
  | 73 \| const entityWarningContent = (

```

![Screenshot 2022-06-24 at 10 57 39 am](https://user-images.githubusercontent.com/10952642/175439406-1df46b81-f032-4b8b-9ad4-0458898bb360.png)

**Comment by kwent** [comment]
Sorry the doc look wrong, can you try 

```js
import { RootlyOverviewCard } from '@rootly/backstage-plugin/src/components/RootlyOverviewCard';
```

**Comment by kwent** [comment]
Oh nevermind will check why `src` didn't make it in

**Comment by kwent** [comment]
Thank you! Should be fixed in [0.1.1](https://www.npmjs.com/package/@rootly/backstage-plugin/v/0.1.1)
